Decred Wallet/Features/Navigation Menu/NavigationMenuViewController.swift

```diff
@@ -306,34 +305,34 @@ extension NavigationMenuViewController: NewBlockNotificationProtocol, NewTransac
         
         switch bestBlockAge {
         case Int64.min...0:
-            self.bestBlockAgeLabel.text = "now"
+            self.bestBlockAgeLabel.text = LocalizedStrings.now
             
         case 0..<Utils.TimeInSeconds.Minute:
-            self.bestBlockAgeLabel.text = "\(bestBlockAge)s ago"
+            self.bestBlockAgeLabel.text = String(format: LocalizedStrings.secondsAgo, bestBlockAge)
             
         case Utils.TimeInSeconds.Minute..<Utils.TimeInSeconds.Hour:
             let minutes = bestBlockAge / Utils.TimeInSeconds.Minute
-            self.bestBlockAgeLabel.text = "\(minutes)m ago"
+            self.bestBlockAgeLabel.text = String(format: LocalizedStrings.minAgo , minutes)
```

Comment:
Please remove the space preceding the comma in this line and similar lines below.